docs/standard/datetime/enumerate-time-zones.md

```diff
@@ -18,7 +18,7 @@ Successfully working with a designated time zone requires that information about
 
 ## To enumerate the time zones present on the local system
 
-1. Call the [TimeZoneInfo.GetSystemTimeZones](xref:System.TimeZoneInfo.GetSystemTimeZones) method. The method returns a generic [ReadOnlyCollection&lt;T&gt;](xref:System.Collections.ObjectModel.ReadOnlyCollection%601) collection of [TimeZoneInfo](xref:System.TimeZoneInfo) objects. The entries in the collection are sorted by their [DisplayName](xref:System.TimeZoneInfo.DisplayName) property. For example:
```

Comment:
this link was correct. you cannot use the url of the API for xref links, you need to use the uid which is found here: https://github.com/docascode/coreapi/